pom.xml

```diff
@@ -84,7 +84,7 @@
         <quarkus-vault.version>4.1.0</quarkus-vault.version>
         <quarkus-operator-sdk.version>6.9.1</quarkus-operator-sdk.version>
 
-        <quarkus-platform-bom-generator.version>0.0.116</quarkus-platform-bom-generator.version>
+        <quarkus-platform-bom-generator.version>0.0.117</quarkus-platform-bom-generator.version>
```

Comment:
@aloubyansky do we need to copy the exclusions here or having them for the core BOM is enough?

Reply:
Having them in the core BOM is enough